src/aks-preview/azext_aks_preview/_validators.py

```diff
@@ -308,6 +308,13 @@ def validate_load_balancer_idle_timeout(namespace):
             raise CLIError(
                 "--load-balancer-idle-timeout must be in the range [4,100]")
 
+def validate_load_balancer_backend_pool_type(namespace):
+    """validate load balancer backend pool type"""
+    if namespace.load_balancer_backend_pool_type is not None:
+        if namespace.load_balancer_backend_pool_type not in ["nodeIP", "nodeIPConfiguration"]:
```

Comment:
Save `["nodeIP", "nodeIPConfiguration"]` in `_consts.py`?

Reply:
done